plugins/test/meson.build

```diff
@@ -5,6 +5,10 @@ if get_option('plugin_dummy')
   install_dummy = true
 endif
 
+install_data(['test-ble.quirk'],
```

Comment:
same comment here - should this have `if get_option('bluez')` around it?

Reply:
yes, this should **certainly** have it; i'll add it now.